apps/condo/domains/acquiring/schema/MultiPayment.js

```diff
@@ -153,7 +155,16 @@ const MultiPayment = new GQLListSchema('MultiPayment', {
         integration: ACQUIRING_INTEGRATION_FIELD,
     },
     hooks: {
-        // TODO (savelevMatthew): Create proper validations
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
```

Comment:
please make sure `resolvedData` is an object, because if not, we can get exception here.

Reply:
It is
<img width="867" alt="image" src="https://user-images.githubusercontent.com/50069872/141249427-de1488f8-b1e7-4ac5-87a3-4f58782b16fd.png">
And hasDvAndSenderFields check dv and sender (ad throws errors) before I access it ;)